packages/react-composites/src/composites/localization/locales/en-US/strings.json

```diff
@@ -47,6 +47,7 @@
     "peopleButtonTooltipContent": "Open People Pane",
     "chatButtonLabel": "Chat",
     "chatButtonTooltipContent": "Open Chat Pane",
+    "chatButtonEnhancedToolTipContent": "new",
```

Comment:
This should just be it's own totally unique string value, something like:
```
chatButtonTooltipContentWithUnreadMessages: "Open Chat Pane - You have new messages! 😲"
```

then tooltip becomes something like:
```
let tooltip = unreadMessages  && strings.chatButtonTooltipContentWithUnreadMessages ||  strings.chatButtonTooltipContent;
```